sharktank/tests/models/llama/llama_cpp_instructions.md

```diff
@@ -0,0 +1,13 @@
+### How to build llama.cpp
+git clone https://github.com/aviator19941/llama.cpp.git
+cd llama.cpp/
+git checkout llama_comparison
+cmake -B build
+cmake --build build --config Release
+
+### How to run llama.cpp
+huggingface-cli download meta-llama/Meta-Llama-3.1-70B --local-dir /home/avsharma/Meta-Llama-3.1-70B
+python convert_hf_to_gguf.py --outtype f16 --outfile Llama-3.1-70B-f16.gguf ../Meta-Llama-3.1-70B/
+
+To predict the prefill token, use `--n-predict 1` and to predict the first decode token, use `--n-predict 2`
+./build/bin/llama-cli -m Llama-3.1-70B-f16.gguf -p "I believe the meaning of life is" --threads 1 --temp 0 --n-predict 1 --no-warmup
```

Comment:
markdown formatting https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

`````suggestion
### How to build llama.cpp
```bash
git clone https://github.com/aviator19941/llama.cpp.git
cd llama.cpp/
git checkout llama_comparison
cmake -B build
cmake --build build --config Release
```

### How to run llama.cpp
```bash
huggingface-cli download meta-llama/Meta-Llama-3.1-70B --local-dir /home/avsharma/Meta-Llama-3.1-70B
python convert_hf_to_gguf.py --outtype f16 --outfile Llama-3.1-70B-f16.gguf ../Meta-Llama-3.1-70B/

To predict the prefill token, use `--n-predict 1` and to predict the first decode token, use `--n-predict 2`
```bash
./build/bin/llama-cli -m Llama-3.1-70B-f16.gguf -p "I believe the meaning of life is" --threads 1 --temp 0 --n-predict 1 --no-warmup
```
`````

You may also want to use portable paths (e.g. `~/` instead of `/home/avsharma/`) and the upstream repo instead of your fork (https://github.com/ggerganov/llama.cpp instead of https://github.com/aviator19941/llama.cpp)